firepit/sqlstorage.py

```diff
@@ -960,6 +960,59 @@ def timestamped(
         else:
             res = qry
         return res
+        
+    def get_all_nested_objects_including_an_attribute_of_SCO(
```

Comment:
Can we come up with a more succinct name?  How about `make_records`?

Also, this function is still nearly identical to `timestamped`.  Can we refactor it to eliminate the code duplication?

Reply:
Hi @pcoccoli,

Thanks for your comment. Let me refactor to eliminate duplicate code. I
will also think about finding a better and more meaningful name.

Regards

Reply:
Hi @pcoccoli , 

We can remove the timestamped function. In Kestrel, we can call the new function (get_all_nested_objects_including_an_attribute_of_SCO()) and pass "first_observed" in the argument. Is it okay with you?

Regards

Reply:
In order to preserve backwards compatibility, can you now add a `timestamped` function that does just that (calls the new function with "first_observed")?
Otherwise, the unit tests will fail.  Also, the existing tests will be covering your change that way.

Reply:
Ok. Sure.